specs/core/0_beacon-chain.md

```diff
@@ -670,8 +670,8 @@ full_deposit_count: uint256
 @payable
 @public
 def deposit(deposit_input: bytes[2048]):
-    assert msg.value >= as_wei_value(MIN_DEPOSIT_AMOUNT, "gwei")
-    assert msg.value <= as_wei_value(MAX_DEPOSIT_AMOUNT, "gwei")
+    assert msg.value >= MIN_DEPOSIT
+    assert msg.value <= MAX_DEPOSIT
 
     index: uint256 = self.deposit_count + TWO_TO_POWER_OF_TREE_DEPTH
     deposit_amount: bytes[8] = slice(concat("", convert(msg.value / GWEI_PER_ETH, bytes32)), start=24, len=8)
```

Comment:
`msg.value / GWEI_PER_ETH` is only "correct" by chance. It should be `msg.value / WEI_PER_GWEI`.